src/Avalonia.Controls/Converters/BottomCornerRadiusFilterConverter.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Globalization;
+using Avalonia.Data.Converters;
+
+namespace Avalonia.Controls.Converters
+{
+    public class BottomCornerRadiusFilterConverter : IValueConverter
```

Comment:
I believe it will be better to port the same converter as WinUI has, because it's not wired to the specific control and can be reused in the Avalonia and be the users:

https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/Common/CornerRadiusFilterConverters.idl
https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/Common/CornerRadiusFilterConverter.h
https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/Common/CornerRadiusFilterConverter.cpp

Reply:
Oh, so *that's* where those converters live in MS's repo...I'll get on that ASAP.


Honestly, this isn't at all how I would address this round corner problem, but...that's another topic, for another Issue/PR/etc.

Reply:
Update: Halfway through porting the WinUI converters, I realized something: These converters operate under the assumption that most (if not all) controls have a `CornerRadius` property.

This assumption seems to be true for WinUI, but is not so for Avalonia.


However, I'm not convinced that following WinUI's example (e.g. adding a `CornerRadius` to every control) is the way to go for Avalonia.

I'll elaborate further in an Issue post shortly, since it's a separate topic, one which may have ramifications well beyond the `Expander` control.

Reply:
https://github.com/AvaloniaUI/Avalonia/discussions/5594 thoughts?

Reply:
@Splitwirez
> These converters operate under the assumption that most (if not all) controls have a CornerRadius property.

Why though? It's simple IValueConverter that has CornerRadius as an input and has some filter/scale properties with returning processed value.

Reply:
Discussion about adding a CornerRadius to Avalonia controls is a good idea.

Reply:
TopLeftValue and BottomRightValue filter kind are strange in WinUI. It doesn't make sense to me right away what these do (ill admit I didnt check the code eithet). If might be best to remove these for Avalonia though.

Reply:
> Why though? It's simple IValueConverter that has CornerRadius as an input and has some filter/scale properties with returning processed value. It doesn't need to operate with these controls directly.

@maxkatz6 Well yes...but that CornerRadius input has to come from somewhere. If not the templated parent, then...where?

Reply:
> Discussion about adding a CornerRadius to Avalonia controls is a good idea.

As explained in my discussion post...while I don't like that idea personally, I think it will probably meet the needs of most developers better than my proposed alternative. Once a final decision is reached, I'd be more than happy to help implement it...even if that means just a `CornerRadius` property and corresponding style/template changes. I guess that should be a separate PR from this one though, correct?

Reply:
> If not the templated parent, then...where?

In WinUI it was done with ThemeResource on older platforms. 
```<Border CornerRadius={Binding Source={StaticResource ControlCornerRadius}, Converter={StaticResource Converter}} />```
I am not sure if DynamicResource will work there though, as it would be preferable.

Reply:
>  I guess that should be a separate PR from this one though, correct?

Yes, "CornerRadius for all controls or something" related questions should be outside of this PR.

Reply:
@maxkatz6 Agreed. I've reached out to you on Discord to discuss how to proceed.